scripts/coverage.sh

```diff
@@ -79,7 +79,7 @@ NPROC=$(nproc)
 JOBS=$((JOBS>NPROC ? NPROC : JOBS))
 
 RUST_LOG=solana=trace _ "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov --no-run "${packages[@]}"
-if RUST_LOG=solana=trace _ "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov "${packages[@]}" -- --nocapture 2> >(tee target/cov/coverage-stderr.log >&2); then
+if RUST_LOG=solana=trace _ ci/upload-console.sh "$cargo" nightly test --jobs "$JOBS" --target-dir target/cov "${packages[@]}" -- --nocapture --color=always; then
```

Comment:
wdyt about dropping `RUST_LOG=solana=trace` by default?  most of the trace output is worthless and dev should debug locally anyway, where they're free to set it

Reply:
hey, careful reviewers. I need to add comment (sorry, no push waiting full ci run...) but this is intentional, otherwise, cov report contains some noise (bunch of `trace!` would be red).

Reply:
oh, duh! also, lame! :joy: